portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
```

Comment:
Hmmm actually this seems to be doing the same thing as the LoginLogoutServlet, did you mean to add both?

Reply:
this servlet is intended to return the status of whether the user is logged in or not, while the loginlogout servlet directs the user to a place where they can login or logout!

Reply:
Ahh got it, I think it's worth clarifying in the javadoc for each class since the names are so similar (maybe rename this one to `loginStatusServlet?`).